infra/aws/terraform/prow-build-cluster/locals.tf

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+locals {
+  root_account_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+
+  configure_prow = var.cluster_name == "prow-build-cluster"
+
+  aws_cli_args = [
+    "eks",
+    "get-token",
+    "--cluster-name",
+    module.eks.cluster_name,
+    "--role-arn",
+    data.aws_iam_role.eks_infra_admin.arn
+  ]
+
+  tags = {
+    Cluster = var.cluster_name
+  }
+
+  auto_scaling_tags = {
+    "k8s.io/cluster-autoscaler/${var.cluster_name}" = "owned"
+    "k8s.io/cluster-autoscaler/enabled"             = true
+  }
+
+  node_group_tags = merge(local.tags, local.auto_scaling_tags)
+
+  azs = slice(data.aws_availability_zones.available.names, 0, 3)
+
+  # Allow cluster admin access to the following IAM roles.
+  cluster_admin_roles = [
+    {
```

Comment:
Let's add a comment for each role in array to what IAM role it is referring to. Example:

```
# This is giving admin access to the EKS infra admin role
...
# This is giving admin access to the EKS cluster admin role
```